database/acid/services.js

```diff
@@ -0,0 +1,23 @@
+const model = require("../models");
```

Comment:
Please convert this to a folder and separate user services from blog services and please encapsulate them into 2 distinct classes!